src/styles/layout.css

```
@@ -27,13 +32,288 @@ code {
+        transition-duration: 0.001ms !important;
+        scroll-behavior: auto !important;
+    }
 }
 
 .max-w-container {
     max-width: 1920px;
     margin: 0 auto;
 }
+
+/* Subtle motion primitives */
+@keyframes floatIn {
```

Stylelint (17.7.0)

[error] Expected keyframe name "floatIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)